config/proposer/loader/loader.go

```diff
@@ -259,18 +262,38 @@ func (psl *settingsLoader) processProposerSettings(loadedSettings, dbSettings *v
 	return newSettings
 }
 
-func processBuilderConfig(current *validatorpb.BuilderConfig, override *validatorpb.BuilderConfig, gasLimitOnly *validator.Uint64) *validatorpb.BuilderConfig {
-	if current != nil {
-		current.GasLimit = reviewGasLimit(current.GasLimit)
-		if override != nil {
-			current.Enabled = override.Enabled
-		}
-		if gasLimitOnly != nil {
-			current.GasLimit = *gasLimitOnly
-		}
+func processBuilderConfig(current *validatorpb.BuilderConfig, options *flagOptions) *validatorpb.BuilderConfig {
+	// If there are no options, return what was passed in
+	if options == nil {
 		return current
```

Comment:
not test covered.

Reply:
in the current setup it's not possible I guess, I can either remove this check but was just trying to be safe